inspire/base/templates/oauthclient/settings/index.html

```diff
@@ -0,0 +1,44 @@
+{#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{%- import "accounts/settings/helpers.html" as helpers with context %}
+{%- extends "accounts/settings/index.html" %}
+
+{% block settings_body %}
+{{helpers.panel_start(
+    'Linked accounts',
+    with_body=False,
+    icon="fa fa-link fa-fw"
+)}}
+<ul class="list-group">
+    {%- for s in services %}
+    <li class="list-group-item">
+    <div class="pull-right">
+        {%- if s.account -%}
+        <a href="{{url_for('oauthclient.disconnect', remote_app=s.appid)}}" class="btn btn-default btn-xs"><i class="fa fa-times-circle"></i> Disconnect</a>
```

Comment:
Once the branch is ready I will deploy on qa.inspirehep.net where we can see better how the oauth page looks like once a user is logged in.

I think we will want to remove the Disconnect button, but you can leave for the moment.